.teamcity/_self/projects/WebApp.kt

```diff
@@ -851,11 +851,28 @@ object PreReleaseE2ETests : BuildType({
 				set -o errexit
 
 				# Retry failed tests only.
-				RETRY_COUNT=1 xvfb-run yarn jest --reporters=jest-teamcity --reporters=default --maxWorkers=%JEST_E2E_WORKERS% --group=calypso-release --onlyFailures
+				RETRY_COUNT=1 xvfb-run yarn jest --reporters=jest-teamcity --reporters=default --maxWorkers=%JEST_E2E_WORKERS% --group=calypso-release --onlyFailures --json --outputFile=pre-release-test-results.json
```

Comment:
The `--json` flag outputs the result of the suite in JSON format, while the `--outputFile` directs Jest to write the aforementioned JSON output to the target file.

Since we're hooking this up to just the Pre-Release Tests for now, I've manually set the file name. When we hook this up to other suites, it should be supplied as a TeamCity managed param.